nodewatcher/nodewatcher.py

```diff
@@ -37,6 +39,7 @@ def getConfig(instance_id):
         logging.getLogger().setLevel(lvl)
     _region = config.get('nodewatcher', 'region')
     _scheduler = config.get('nodewatcher', 'scheduler')
+    _idle_time = config.get('nodewatcher', 'idle_time')
```

Comment:
I don't like the name `idle_time`, it should be something closer to `cooldown_period`

Reply:
There is already a cooldown parameter and adding something similar would causing confusion. Let's stick with idle time or rename the other one and change this to cooldown_period.

scaling_cooldown
Amount of time in seconds to wait before attempting further scaling actions.

Defaults to 300 for the default template.

scaling_cooldown = 300

Reply:
I agree _cooldown_ would be confusing in this case and even if we are going to remove it from the new scale-up strategy we should preserve some consistency with the terminology used until now.